modules/auxiliary/server/browser_autopwn.rb

```diff
@@ -236,9 +236,9 @@ def setup
       print_debug("NOTE: Debug Mode; javascript will not be obfuscated")
     else
       pre = Time.now
-      print_status("Obfuscating initial javascript #{pre}")
-      @init_js.obfuscate
-      print_status "Done in #{Time.now - pre} seconds"
+      #print_status("Obfuscating initial javascript #{pre}")
```

Comment:
I guess you're commenting this code because obfuscation is wasting something, right? Do you mind to add a comment explaining what is going on so others reading this code will know why this code is commented.